etc/kayobe/opensm.yml

```diff
@@ -4,7 +4,6 @@
 
 # Whether OpenSM is enabled.
 #opensm_enabled:
-opensm_enabled: True
```

Comment:
Intentional disabling of OpenSM?

Reply:
Yes - the controller isn't on the IB network, so we're running a subnet manager on the IB switch.